packages/frontend-v2/components/Plan/PlanConcentrationsSelect.tsx

```diff
@@ -26,20 +22,16 @@ export const PlanConcentrationsSelect: React.FC<
 
   const isConcentrationRequired = supportedMajor.minRequiredConcentrations > 0;
```

Comment:
I think it might make sense to allow concentrations to be optional in a similar line of reasoning to allowing a major to be optional. This is probably outside the scope of this PR for the moment though, so it might be worth opening an issue for it.

Reply:
def, already have an issue open for it!